docs/differences_with_venv.rst

```diff
@@ -0,0 +1,17 @@
+Differences with venv
+=====================
+
+- ``venv`` is bundled with Python itself (starting from 3.3+), ``virtualenv``
+  is a third party package. This allows for ``virtualenv`` to have greater
+  flexibility when dealing with several Python versions.
+- ``virtualenv`` allows further :doc:`extendibility <extend>` to override default
+  functionality through a plugin system.
+- ``virtualenv`` can create virtual environments through its interpreter discovery
+  technique from arbitrary Python locations, they do not be installed or be on the
```

Comment:
This is not true.

Reply:
This also comes naively from your presentation https://gaborbernat.github.io/euro_python_2019/#/

But I could be wrong on the "installation" part.